src/cli.ts

```diff
@@ -1,51 +1,537 @@
+      defaultDescription: 'os language at runtime of the app',
+      description:
+        'set the language or locale to render the web site as (e.g., "fr", "en-US", "es", etc.)',
+      group: 'Internal Browser Options',
+      type: 'string',
+    })
+    .option('u', {
+      alias: 'user-agent',
+      description: 'set the app user agent string',
+      group: 'Internal Browser Options',
+      type: 'string',
+    })
+    // URL Handling Options
+    .option('block-external-urls', {
+      default: false,
+      description: `forbid navigation to URLs not considered "internal" (see '--internal-urls').  Instead of opening in an external browser, attempts to navigate to external URLs will be blocked`,
+      group: 'URL Handling Options',
+      type: 'boolean',
+    })
+    .option('internal-urls', {
+      defaultDescription: 'URLs sharing the same base domain',
+      description:
+        'regex of URLs to consider "internal"; all other URLs will be opened in an external browser.',
+      group: 'URL Handling Options',
+      type: 'string',
+    })
+    .option('proxy-rules', {
+      description:
+        'proxy rules; see https://www.electronjs.org/docs/api/session#sessetproxyconfig',
+      group: 'URL Handling Options',
+      type: 'string',
+    })
+    // Auth Options
+    .option('basic-auth-password', {
+      description: 'basic http(s) auth password',
+      group: 'Auth Options',
+      type: 'string',
+    })
+    .option('basic-auth-username', {
+      description: 'basic http(s) auth username',
+      group: 'Auth Options',
+      type: 'string',
+    })
+    // Graphics Options
+    .option('disable-gpu', {
+      default: false,
+      description: 'disable hardware acceleration',
+      group: 'Graphics Options',
+      type: 'boolean',
+    })
+    .option('enable-es3-apis', {
+      default: false,
+      description: 'force activation of WebGL 2.0',
+      group: 'Graphics Options',
+      type: 'boolean',
+    })
+    .option('ignore-gpu-blacklist', {
+      default: false,
+      description: 'force WebGL apps to work on unsupported GPUs',
+      group: 'Graphics Options',
+      type: 'boolean',
+    })
+    // Caching Options
+    .option('clear-cache', {
+      default: false,
+      description:
+        'prevent the application from preserving cache between launches',
+      group: 'Caching Options',
+      type: 'boolean',
+    })
+    .option('disk-cache-size', {
+      defaultDescription: 'chromium default',
+      description:
+        'forces the maximum disk space (in bytes) to be used by the disk cache',
+      group: 'Caching Options',
+      type: 'number',
+    })
+    // (In)Security Options
+    .option('disable-old-build-warning-yesiknowitisinsecure', {
+      default: false,
+      description:
+        'Disables warning when opening an app made with an old version of Nativefier. Nativefier uses the Chrome browser (through Electron), and it is dangerous to keep using an old version of it.)',
```

Comment:
```suggestion
        'disables warning when opening an app made too long ago. Nativefier uses the Chrome browser (through Electron), and it is dangerous to keep using an old version of it',
```

Reply:
> Here and everywhere: please make a pass of homogenizing:
> 
> 1. The tense of descriptions. Currently it is a mix of infinitive ("ignore", "make"), and singular 3rd person ("disables")
> 2. Capitalization of `description`'s first character. Currently, some descriptions start capitalized, some don't.
> 
> I don't have a preference; up to you to see what fits best.

No problem. In this case I was trying to keep the existing verbiage, but I'll look at cleaning it up.